venia-integration-tests/run-tests

```diff
@@ -17,5 +17,18 @@ PR_INSTANCE_URL=$1
 # collect update screenshots flag
 UPDATE_SCREENSHOTS=${2:-false}
 
-# run cypress tests and remove the container once done
-docker run --rm -v $PWD:/venia-integration-tests -w /venia-integration-tests --entrypoint=cypress cypress/included:8.3.1 run --browser chrome --config baseUrl=$PR_INSTANCE_URL,screenshotOnRunFailure=false --config-file cypress.config.json --env updateSnapshots=$UPDATE_SCREENSHOTS --headless
+# see if its a local instance or a pr instance
+PORT="$(echo $1 | sed -e 's,^.*:,:,g' -e 's,.*:\([0-9]*\).*,\1,g' -e 's,[^0-9],,g')"
```

Comment:
This change is from #3422 and it will be gone when it gets merged into `develop`. @michaelyu0123 had to base his branch off of #3422 to have the ability to run tests off of a local instance.

Reply:
well if you push this change to revanth/cypress_docker_local branch I can sync it.